.gitignore

```diff
@@ -21,3 +20,9 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+
+.eslintrc.js
+
+# dev profile image
+/src/components/assets/images/avatar.jpg
```

Comment:
If you gitignore this file but still use it in the code then a reviewer running the branch gets the following error:
`Module not found: Error: Can't resolve '../assets/images/avatar.jpg' in '.../src/components/Profile'`